Sources/HTMLEntities/String+HTMLEntities.swift

```diff
@@ -0,0 +1,338 @@
+/*
+ * Copyright IBM Corporation 2016
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/// Generated from the list of HTML4 entities here:
+/// https://www.w3.org/TR/html4/sgml/entities.html
+
+#if os(Linux)
```

Comment:
I tried it, but it doesn't seem to be possible to inject the OS checks inside the definition of a dictionary.